tests/README.md

```diff
@@ -88,28 +88,27 @@ Run using
 This test compares the stdlib stubs against the objects at runtime. Because of
 this, the output depends on which version of Python and on what kind of system
 it is run.
-Thus the easiest way to run this test is via Github Actions on your fork;
-if you run it locally, it'll likely complain about system-specific
-differences (in e.g, `socket`) that the type system cannot capture.
+As such, if you run this test locally, it *may* complain about system-specific
+differences (in e.g, `socket`) that the type system has difficulty capturing.
+If you run into this issue, you can consider
+[running the test via Github Actions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow)
+on your typeshed fork. This ensures that the test is run in an environment
+that exactly matches typeshed's CI.
+
 If you need a specific version of Python to repro a CI failure,
-[pyenv](https://github.com/pyenv/pyenv) can help.
+[pyenv](https://github.com/pyenv/pyenv) can also help.
 
-Due to its dynamic nature, you may run into false positives. In this case, you
-can add to the allowlists for each affected Python version in
-`tests/stubtest_allowlists`. Please file issues for stubtest false positives
+Stubtest is an imperfect tool, and sometimes produces false-positive errors.
```

Comment:
This implies that false positives are generally due to stubtest imperfections. With the original text I was going for "things are dynamic so we can't get always get things right, it's fine to allowlist and move on" as opposed to "file bugs on stubtest if you get some false positive" (or "give up and be confused"). Again, not quite sure what the change in wording is meant to help with.

Reply:
This was meant to just be some incidental wordsmithing, but you raise good points; the changes here aren't very well motivated. I'll revert the changes to this section for now.